feat(i18n): add zh-TW (Traditional Chinese) support

muramato opened the pull request:
## Change Description
- Added the app/I18N/zh_TW/ directory and associated files (Arms.php, Chest.php, Head.php, etc.);
- Added Traditional Chinese (zh-TW) localization translations, tailored to the terminology conventions of Taiwanese Traditional Chinese for game equipment and items;
- The translation format is consistent with the existing en-GB and ja-JP versions, with no code formatting errors.
## Testing Notes
Local verification results:
- After switching the language to zh-TW, the names of the vast majority of equipment/items are displayed correctly in Traditional - Chinese.
- Translations are referenced from the official Traditional Chinese terminology of Monster Hunter and the file [mhf internal ids.xlsx](https://github.com/user-attachments/files/24362974/mhf.internal.ids.xlsx);
- There are no code conflicts, and this PR can be merged directly.
---
## 变更说明

- 新增 `app/I18N/zh_TW/` 目录及相关文件（Arms.php/Chest.php/Head.php 等）；
- 补充繁体中文的国际化翻译，适配游戏装备/道具的台繁用语习惯；
- 翻译格式与现有en-GB、ja-JP保持一致，无代码格式错误。

## 测试说明
本地验证：
- 切换语言为 zh-TW 后，绝大多数装备/道具名称均正常显示繁体中文。

## 其他说明
- 翻译参考《怪物猎人》官方繁体用语和[mhf internal ids.xlsx](https://github.com/user-attachments/files/24362974/mhf.internal.ids.xlsx)
- 无代码冲突，可直接合并。

Commit:
feat(i18n): add zh-TW (Traditional Chinese) support

Chakratos commented on app/I18N/zh_TW/PoogieOutfits.php:
```diff
@@ -0,0 +1,32 @@
+[
```
There seems to be a missing:
`<?php
return [`

in this file.

Chakratos commented:
English

Hi @muramato, thank you for adding Traditional Chinese (zh-TW) support — much appreciated.

I ran through the changes and there are two small fixes needed before we can merge:

PoogieOutfits.php needs the standard PHP wrapper
Right now it starts with [ and ends with ]; (so it’s missing <?php and the return ...;). Please update it to match the format used by the other locale files, e.g. <?php return [ ... ];.

Add a trailing newline
GitHub indicates “No newline at end of file” for that file. Please add a final newline to keep formatting consistent.

Once that’s updated, I’ll take another look and we should be good to go. Thanks again.

繁體中文（Traditional Chinese）

嗨 @muramato，感謝你加入繁體中文（zh-TW）的支援，真的很感謝你的貢獻。

我看過這次的變更了，合併前只需要修正兩個小地方：

PoogieOutfits.php 需要補上標準的 PHP 格式
目前檔案是直接以 [ 開頭、以 ]; 結尾，缺少 <?php 與 return ...;。請調整成和其他語系檔一致的格式，例如 <?php return [ ... ];。

補上檔案結尾換行
GitHub 顯示該檔案 “No newline at end of file”，請在檔案最後補上一個換行，讓格式更一致。

你更新後我會再確認一次，應該就可以合併了。再次感謝。

muramato committed:
fix(i18n): Fix translations and code formatting/修正翻译和代码格式

- Correct and complete translations based on community edition client/基于社区版客户端修正、补全翻译内容
- Adjust code formatting to comply with specification requirements/调整代码格式以符合规范要求

muramato commented:
The Traditional Chinese (zh-TW) support  updates have been completed.

muramato committed:
fix(i18n): Fix Traditional Chinese item name translations / 修正繁體中文物品名稱翻譯

- Update AA00 item name / 更新 AA00 物品名稱為「血塗的落暉玉」
- Update AB00 item name / 更新 AB00 物品名稱為「不羈的願物玉」
- Add 5525 item name / 新增 5525 物品名稱為「RD太刀生産券」
- Add 5625 item name / 新增 5625 物品名稱為「RB太刀生産券」
- Add 5725 item name / 新增 5725 物品名稱為「圓月Ｅ生産券」
- Add 5825 item name / 新增 5825 物品名稱為「圓月Ｄ生産券」
